app/src/main/java/com/cornellappdev/android/eatery/CustomMapView.java

```diff
@@ -0,0 +1,37 @@
+package com.cornellappdev.android.eatery;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.MotionEvent;
+
+import com.google.android.gms.maps.GoogleMapOptions;
+import com.google.android.gms.maps.MapView;
+
+// This class will allow a lot more customization for the mapview, and is necessary if we want to
+// add in scrolling or change the map functionality more
+public class CustomMapView extends MapView {
+
+    public CustomMapView(Context context) {
```

Comment:
can you add comments above each constructor explaining the difference between each one? it doesn't seem like all the different constructors are used in this pr.

Reply:
I was planning on leaving this in in case we needed features later but it is pretty unnecessary. I'm just gonna delete this class

Reply:
good idea! 🙅